index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/mobile.css">
+    <link rel="stylesheet" href="styles/desktop.css">
+    <title>Garvin Chimone</title>
+</head>
+<body>
+    <nav>
+        <div class="title"><h2>My Logo</h2></div>
+        <div class="burger-menu">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>
+    </nav>
```

Comment:
- [ ] Great job on implementing the first face of your portfolio project as specified in the project requirement! 👏 However, there are some minor UI issues 🐞 that we need to address and fix.

  - [ ] Consider wrapping your header section using a `header` tag for improved `semantic` structure. The `header` tag is specifically designed to mark the introductory content or a container for introductory content on a webpage. Using the `header` tag will enhance the clarity and maintain the correct `semantic` meaning of your header section.